issuers/vault/vault.go

```diff
@@ -38,6 +38,10 @@ type Issuer struct {
 	// TimeToLive configures the lifetime of certificates
 	// requested from the Vault server.
 	TimeToLive time.Duration
+
+	// URISubjectAlternativeNames defines custom URI SANs.
```

Comment:
Could you add a little blurb about what format these URIs take? A link to the vault docs would be great, together with a little example.

Reply:
Unfortunately there's not a good spot in the vault docs about this field other than linking to the full docs for the sign endpoint. I've written in the relevant information in the comment though.